### PR TITLE
Update databases

### PR DESCRIPTION
**Pull request by sergiobenrocha2**
rdb files built again with dat in the end

### CODE REVIEW

**Comment by RobLoach** [comment]
:+1:

File: dat/Nintendo - Game Boy Advance.dat [hunk 1]
```diff
@@ -9921,7 +9921,7 @@ game (
 
 game (
 	name "Mega Man Battle Network 3 - Blue Version (USA) (Virtual Console)"
-	description "Mega Man Battle Network 3 - Blue Version (USA) (Virtual Console) (From Wii U DL "Mega Man Battle Network 3 - Blue Version (USA) (GBA) (Virtual Console)")"
+	description "Mega Man Battle Network 3 - Blue Version (USA) (Virtual Console) (From Wii U DL 'Mega Man Battle Network 3 - Blue Version (USA) (GBA) (Virtual Console)')"
```
Comment (RobLoach) [inline dat/Nintendo - Game Boy Advance.dat]: The latest Dat-O-Matic update fixes this.

**Comment by RobLoach** [comment]
My recommendation would be to:

1. git reset to commit `884a4ca`
2. Update the GBA dat again so that you have their latest fix
3. Keep RDB updates out of DAT updates just to make tracking easier

**Comment by sergiobenrocha2** [comment]
Ok, there.

@RobLoach , is PC-FX dat ready?

**Comment by RobLoach** [comment]
> is PC-FX dat ready?

The majority of the optical media DATs were updated in other PRs with https://github.com/robloach/libretro-dats :wink:

**Comment by sergiobenrocha2** [comment]
Wow, cool.

I'm done with the updates.

**Comment by RobLoach** [comment]
Nicely done! This looks good, :+1:  to merge. We can update the RDBs after a few of the other PRs are addressed.

**Pull request merged**